### PR TITLE
Add Windows references and screenshots to Write your first flutter app

### PR DESCRIPTION
**Pull request by miquelbeltran**
@RedBrogdon @domesticmouse 

In this Draft I want to show you the possible changes we can do to the "Write your first Flutter app" codelab on the website. These changes are not yet done on the docs codelab version, neither on the part 2.

The codelab code is not changed.

Specially I want to check the screenshots, if this is something you like or if you think there's a better way.

Example:
image

I really don't want to mess with the CSS :) but I guess I will have to figure out how to give a bit of spacing between the two. This is a modification of the Android & iOS screenshot template.

Here's a live version of the changes in the PR: https://mb-flutter-dev-1.web.app/get-started/codelab

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by domesticmouse** [comment]
As an initial comment on the screenshot in the PR text, I'd narrow the windows app by re-sizing it. The design of the app itself assumes a mobile phone shaped display, and thus looks terrible in landscape or square windows.

Thoughts @RedBrogdon and @sfshaza2 ?

**Comment by miquelbeltran** [comment]
I can resize the window to 350 x 689, which is the same size as the iOS screenshot, in that case the app would look like this

image

Would that be OK?

File: src/get-started/codelab.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The animated GIF shows how the app works at the completion of part 1.
   <h4 class="no_toc">What you’ll learn in part 1</h4>
 
   * How to write a Flutter app that looks natural on iOS, Android,
-    and the web
+    Windows, and the web
```
Comment (domesticmouse) [inline src/get-started/codelab.md:59]: @RedBrogdon any particular reason this should be only windows? I'd be happy to mention all six platforms here.

File: src/get-started/codelab.md [hunk 2]
```diff
@@ -99,6 +100,14 @@ other browsers. For more information,
 see [Building a web application with Flutter][]
 and [Write your first Flutter app on the web][codelab-web].
 
+Also, Flutter apps can compile for desktop.
+When you are developing Flutter apps in Windows,
```
Comment (domesticmouse) [inline src/get-started/codelab.md:104]: Consider re-working this to be about all three desktop OS platforms.

**Review by domesticmouse — APPROVED** [review]
I'd be very tempted to grow this from just windows to all three desktop OS platforms.

**Comment by miquelbeltran** [comment]
image

Added a 32px padding for windows screenshots

File: src/get-started/codelab.md (hunk 2 above)
Comment (miquelbeltran) [reply src/get-started/codelab.md]: I reworded to make it more generic

File: src/get-started/codelab.md (hunk 1 above)
Comment (miquelbeltran) [reply src/get-started/codelab.md]: Also reworded this section to make it more generic

**Comment by domesticmouse** [comment]
PTAL @sfshaza2

**Comment by miquelbeltran** [comment]
I have added a couple of extra references to Linux and macOS, and proceed to modify the codelab docs with the same changes.

File: src/get-started/codelab.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ The animated GIF shows how the app works at the completion of part 1.
   <h4 class="no_toc">What you’ll learn in part 1</h4>
 
   * How to write a Flutter app that looks natural on iOS, Android,
-    and the web
+    desktop (e.g. Windows), and the web
```
Comment (sfshaza2) [inline src/get-started/codelab.md:59]: e.g. is against the style guide  :)

```suggestion
    desktop (Windows, for example), and the web
```

**Review by sfshaza2 — APPROVED** [review]
One minor nit, but otherwise looks great!

**Comment by domesticmouse** [comment]
Thanks @miquelbeltran!

**Comment by miquelbeltran** [comment]
@domesticmouse I have applied the Shams' change in the Google Docs codelab as well, if you want to publish those

**Comment by sfshaza2** [comment]
You rock, @miquelbeltran! Thx!

**Comment by domesticmouse** [comment]
> @domesticmouse I have applied the Shams' change in the Google Docs codelab as well, if you want to publish those

Thanks again @miquelbeltran!

For the googlers in the room, cl/427945938

**Pull request merged**